diff --git a/powerline_shell/__init__.py b/powerline_shell/__init__.py
index ae62704..c043a11 100644
--- a/powerline_shell/__init__.py
+++ b/powerline_shell/__init__.py
@@ -11,7 +11,7 @@ import re
 
 
 def _current_dir():
-    """Returns the full current working directory as the user would have used
+    r"""Returns the full current working directory as the user would have used
     in their shell (ie. without following symbolic links).
 
     With the introduction of Bash for Windows, we can't use the PWD environment
diff --git a/powerline_shell/segments/git.py b/powerline_shell/segments/git.py
index de2b049..814bad4 100644
--- a/powerline_shell/segments/git.py
+++ b/powerline_shell/segments/git.py
@@ -4,7 +4,7 @@ from ..utils import RepoStats, ThreadedSegment, get_git_subprocess_env
 
 
 def parse_git_branch_info(status):
-    info = re.search('^## (?P<local>\S+?)''(\.{3}(?P<remote>\S+?)( \[(ahead (?P<ahead>\d+)(, )?)?(behind (?P<behind>\d+))?\])?)?$', status[0])
+    info = re.search(r'^## (?P<local>\S+?)'r'(\.{3}(?P<remote>\S+?)( \[(ahead (?P<ahead>\d+)(, )?)?(behind (?P<behind>\d+))?\])?)?$', status[0])
     return info.groupdict() if info else None
 
 
